creme/creme_core/forms/enumerable.py

```diff
@@ -245,6 +245,11 @@ def default_url(self):
         )
 
 
+# Set the default debounce delay to 300ms to reduce the number of API calls.
+# The default value in the JS component is 100ms.
+ENUMERABLE_DEFAULT_DEBOUNCE_DELAY = 300
```

Comment:
What do you think about a class attribute, so the value could be changed, on all instances, on a particular sub-class' instances, or on a particular instance ?

Reply:
Sounds good!